package.json

```diff
@@ -5,7 +5,7 @@
   "description": "A GitHub Action to send arbitrary requests to GitHub's REST API",
   "main": "dist/index.js",
   "scripts": {
-    "build": "ncc build index.js -o dist",
+    "build": "esbuild --bundle --platform=node --target=node20 --outfile=dist/index.js index.js",
```

Comment:
We need to validate that assets are still getting persisted properly.

Reply:
I can handle in a worklow if the checked in bundled code is valid or not.